### ISSUE TITLE
feat: Automatically detect discord emoji IDs and respond with sticker

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2033]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.17.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.16.0...4.17.0) (2021-08-20)

- build(deps-dev): bump tsc-watch from 4.4.0 to 4.5.0 [`#2031`](https://github.com/open-wa/wa-automate-nodejs/pull/2031)
- build(deps-dev): bump typedoc from 0.21.5 to 0.21.6 [`#2032`](https://github.com/open-wa/wa-automate-nodejs/pull/2032)
- build(deps-dev): bump @types/node from 16.6.1 to 16.6.2 [`#2027`](https://github.com/open-wa/wa-automate-nodejs/pull/2027)
- build(deps-dev): bump ts-node from 10.2.0 to 10.2.1 [`#2028`](https://github.com/open-wa/wa-automate-nodejs/pull/2028)
- Release 4.17.0 [`7c6ce51`](https://github.com/open-wa/wa-automate-nodejs/commit/7c6ce5121a650fd0461b6c6bee0b01007bfa16a5)
- ✨ feat: `client.sendEmoji` #2033 [`749a0e4`](https://github.com/open-wa/wa-automate-nodejs/commit/749a0e4d8ea3eedeaeb23721d2821416b3c74ac2)
- ✨ feat: `config.autoEmoji` set ident char (default `:`) #2033 [`f68d16b`](https://github.com/open-wa/wa-automate-nodejs/commit/f68d16bc4cf009eff9e27d450e7e05c53072dde8)
- 🥚  patches update [`c1ece12`](https://github.com/open-wa/wa-automate-nodejs/commit/c1ece129f9eed1dc36c850c5c241b4045d82b514)
- 🥚 patches update [`6266fca`](https://github.com/open-wa/wa-automate-nodejs/commit/6266fcadd65504173f4a0bc654cd101c993aa519)
- 🥚 patches update [`fa039a7`](https://github.com/open-wa/wa-automate-nodejs/commit/fa039a7916640a8c698dcdee7c076e11ddeebdca)
- 📝  fix config object link in readme #2024 [`b3fce74`](https://github.com/open-wa/wa-automate-nodejs/commit/b3fce74116e480a207ffbf7f52a3f7c657ba0abf)
